src/style.css

```diff
@@ -1 +1,65 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
```

Comment:
you shouldn't reset margins in the * selector

Reply:
still not fixed
Add it to html tags directly(where you need it)